main.py

```diff
@@ -71,19 +64,30 @@ def showMap(city):
 
 @app.route('/', methods=['GET', 'POST'])
 def index():
-    cityName = getRandomCityName(POPULATION_THRESHHOLD)
-    cityCoords = getCityCoordinatesByName(cityName)
-    city = City(cityName, cityCoords)
+    city = getRandomCity()
     mapOfCity = showMap(city)
+    correctAnswer = city.cityId
+
+    return render_template('index.html', mapOfCity=mapOfCity, correctAnswer=correctAnswer)
+
+@app.route('/guess', methods=['POST'])
+def checkAnswer():
```

Comment:
`guessController` or something like that. Make the name somewhat similar to the route youre making it imo